utils/random_string_generator.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"crypto/rand"
+)
+
+// GetRandomString returns a string of a given length
+func GetRandomString(length int) string {
```

Comment:
You may use go pkg for random generation of string, or a similar way https://github.com/litmuschaos/litmus-go/blob/f3997aa2dd0de8a76dfc25c1ef9b485cdf484f35/pkg/utils/common/common.go#L53

Reply:
I think we can use this:
```go
// GetRandomString generate a random string
func GetRandomString() string {
	var letterRunes = []rune("abcdefghijklmnopqrstuvwxyz")
	runID := make([]rune, 6)
	rand.Seed(time.Now().UnixNano())
	for i := range runID {
		runID[i] = letterRunes[rand.Intn(len(letterRunes))]
	}
	return string(runID)
}
```
@oumkale was pointing to this I think.